zigate/mqtt_broker.py

```diff
@@ -49,9 +49,11 @@ def _publish(self, topic, payload=None):
     def device_changed(self, device):
         logging.debug('device_changed {}'.format(device))
         self._publish('zigate/device_changed/{}'.format(device.addr), device)
+        self._publish('zigate/device_missing/{}'.format(device.addr), device.missing)
 
     def device_removed(self, addr):
         logging.debug('device_removed {}'.format(addr))
+        self._publish('zigate/device_missing/{}'.format(addr), True)
```

Comment:
You can't publish device_missing on device_removed, it makes no sense since the device is removed...